App/Services/CredentialManager.cs

```diff
@@ -223,8 +223,9 @@ private async Task<CredentialModel> LoadCredentialsInner(CancellationToken ct)
             };
         }
 
-        // Grab the lock again so we can update the state.
-        using (await _opLock.LockAsync(ct))
+        // Grab the lock again so we can update the state. Don't use the CT
+        // here as it may have already been canceled.
+        using (await _opLock.LockAsync(TimeSpan.FromSeconds(5), CancellationToken.None))
```

Comment:
This was a bug I noticed on startup when the server was unreachable, it would timeout the CT and then never set the status to "invalid" so the tray window would always be stuck in the loading state